src/main/java/log_out/interface_adapters/LogOutController.java

```diff
@@ -10,14 +10,17 @@ public class LogOutController {
 
     private final LogOutInputBoundary interactor;
 
-
+    /**
+     * initialize a LogOutController Instance
+     * @param inputBoundary input boundary for the use case
+     */
     public LogOutController(LogOutInputBoundary inputBoundary){
         this.interactor = inputBoundary;
     }
 
     //Initializes the logOut Use case
-    public void logOutInitializer(LogOutControllerData dataFromController){
-        LogOutRequestModel requestModel = new LogOutRequestModel(dataFromController.getLogOutRequest());
+    public void logOutInitializer(LogOutUserInputData dataFromController){
```

Comment:
Is the InputData that's being passed in here being used?

Reply:
it was a mistake, thanks